components/script/dom/webidls/PerformanceResourceTiming.webidl

```diff
@@ -23,6 +23,7 @@ interface PerformanceResourceTiming : PerformanceEntry {
     readonly attribute DOMHighResTimeStamp requestStart;
     readonly attribute DOMHighResTimeStamp responseStart;
     readonly attribute DOMHighResTimeStamp responseEnd;
+    readonly attribute DOMHighResTimeStamp duration;
```

Comment:
Thanks for pointing out the difference with the specification - duration is actually a property of PerformanceEntry, which PerformanceResourceTiming inherits from, so the specification points out that we should be ensuring that that interface's Duration method returns the difference between ResponseEnd and StartTime.